CMakeLists.txt

```diff
@@ -1,43 +1,62 @@
-cmake_minimum_required (VERSION 3.2.2)
+cmake_minimum_required (VERSION 3.4.3)
```

Comment:
Check with @mehdi-goli to see whether we can do this or not. IIRC there's a board that we want to test on that doesn't support new(ish) cmakes (it is definitely below 3.6.0), but it could be that it supports up to version 3.4.3.

Reply:
@AdamHarries yes, this version is OK